src/core/search/indices.cc

```diff
@@ -88,18 +94,24 @@ const CompressedSortedSet* BaseStringIndex::Matching(string_view str) const {
   else
     word = una::cases::to_lowercase_utf8(str);
 
-  auto it = entries_.find(word);
+  auto* mr = entries_.get_allocator().resource();
+  auto it = entries_.find(pmr::string{word, mr});
   return (it != entries_.end()) ? &it->second : nullptr;
```

Comment:
pmr strings are a PAIN there is NO comparison operator for them and string/string_view

what is more, they weight one pointer more

I'm thinking about creating a ConstString that will store only the chars / an inlined buffer

Reply:
Another quicker solution would be to create a wrapper type over pmr::string that will allow implicit conversion to string_view so find will work without an intermediary object

Reply:
Instead of a wrapper, a less scary solution could be a custom `operator==`